### PR TITLE
Add some json-schema example

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

n/a

**Description**

Add more complex exemple with json schema

**Additional context**

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease
```
npm install @gravitee/ui-policy-studio-angular@5.7.3-json-schema-d2114c2
```
```
yarn add @gravitee/ui-policy-studio-angular@5.7.3-json-schema-d2114c2
```

---

### CODE REVIEW

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/testing-json-schema/endpoints.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const endpointsGetResponse = [
...
...                   "properties": { "type": { "pattern": "JKS|PKCS12" } },\n                            "required": ["content", "password"]\n                        },\n                        {\n                            "properties": { "type": { "pattern": "JKS|PKCS12" } },\n                            "required": ["path", "password"]\n                        }\n                    ]\n                }\n            }\n        }\n    },\n    "required": ["target"],\n    "additionalProperties": false,\n    "x-schema-form": {\n        "errors": {\n            "proxy": {\n                "oneOf": "The host and port are <span class=\\"error\\">required</span>"\n            },\n            "ssl": {\n                "trustStore": {\n                    "oneOf": "A path or a content is <span class=\\"error\\">required</span> - for JKS and PKCS#12 a password is also <span class=\\"error\\">required</span>"\n                },\n                "keyStore": {\n                    "oneOf": "Paths or contents are <span class=\\"error\\">required</span> - for JKS and PKCS#12, a password is also <span class=\\"error\\">required</span>."\n                }\n            }\n        }\n    }\n}\n',
```
Comment (loriepisicchio) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/testing-json-schema/endpoints.ts:30]: Maybe the file header needs to be updated ?

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/type-component/array-type.component.ts [hunk 2]
```diff
@@ -19,8 +19,8 @@ import { FieldArrayType } from '@ngx-formly/core';
 @Component({
   selector: 'gio-fjs-array-type',
   template: `
-    <div>
-      <legend *ngIf="to.label">{{ to.label }}</legend>
+    <div class="wrapper">
+      <h4 *ngIf="to.label">{{ to.label }}</h4>
```
Comment (loriepisicchio) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/type-component/array-type.component.ts:23]: For accessibility reason, we should not use `h4` tags for styling only. Maybe we can replace this with 
```suggestion
      <div class="mat-subheading-1" *ngIf="to.label">{{ to.label }}</div>
```

**Review by loriepisicchio — APPROVED** [review]

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/testing-json-schema/endpoints.ts (hunk 1 above)
Comment (ThibaudAV) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/testing-json-schema/endpoints.ts:30]: ha no, we have to revert, it's an attempt with a external validator. and the current schema is not valid with last JsonChema standard

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/type-component/array-type.component.ts (hunk 2 above)
Comment (ThibaudAV) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/type-component/array-type.component.ts:23]: 👌 
we just have to keep the margin underneath or I have to add it

**Review by gcusnieux — APPROVED** [review]

**Pull request merged**